### PR TITLE
Replaced fa-tag icons with Tag

### PR DESCRIPTION
**Pull request by NidhiKJha**
## Explanation

Replaced instances of `fa-tag` with TAG icons. 

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in the browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by NidhiKJha** [comment]
@georgewrmarshall, the tag icon we have in the `component-library` looks different from what fa-tag.

[Current tag](https://metamask.github.io/metamask-storybook/?path=/story/components-app-approvecontentcard--default-story)

[New Tag Icon](https://metamask.github.io/metamask-storybook/?path=/story/components-componentlibrary-icon--default-story)

**Comment by NidhiKJha** [comment]
@georgewrmarshall, the tag icon we have in the `component-library` looks different from fa-tag.

[Current tag](https://metamask.github.io/metamask-storybook/?path=/story/components-app-approvecontentcard--default-story)

[New Tag Icon](https://metamask.github.io/metamask-storybook/?path=/story/components-componentlibrary-icon--default-story)

**Comment by georgewrmarshall** [comment]
> 

Yup! It's good to go :D

File: ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js [hunk 1]
```diff
@@ -26,7 +26,11 @@ import LedgerInstructionField from '../../../components/app/ledger-instruction-f
 import { TokenStandard } from '../../../../shared/constants/transaction';
 import { CHAIN_IDS, TEST_CHAINS } from '../../../../shared/constants/network';
 import ContractDetailsModal from '../../../components/app/modals/contract-details-modal/contract-details-modal';
-import { ICON_NAMES, ButtonIcon } from '../../../components/component-library';
+import {
+  ICON_NAMES,
+  ButtonIcon,
```
Comment (georgewrmarshall) [inline ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js:31]: What are you using `ButtonIcon` for here?

**Review by georgewrmarshall — COMMENTED** [review]
Looking good. One comment. I know this is a lot but to give me full confidence in PR changes I would love to see screenshots of at least the new icons in place :)

File: ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js (hunk 1 above)
Comment (NidhiKJha) [reply ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js:31]: No, I haven't added ButtonIcon here instead I imported `Icon` here. In this file, `ButtonIcon` is used for `copy` icon. This PR is to update the tag icon only, hence I didn't update the Copy Icon here.

Comment (georgewrmarshall) [reply ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js:31]: Oh duh it was already imported mb 🤦

**Review by georgewrmarshall — APPROVED** [review]
LGTM

**Review by darkwing — APPROVED** [review]

**Pull request merged**